users/strelaysrv.rst

```diff
@@ -77,6 +79,80 @@ Options
 
     Listen address for status service (blank to disable) (default ":22070").
 
+
+Setting Up
+----------
+
+Primarily, you need to decide on a directory to store the TLS key and
+certificate and a listen port. The default listen port of 22067 works, but for
+optimal compatibility a well known port for encrypted traffic such as 443 is
+recommended. This may require additional setup to work without running
+as root or a privileged user, see `Running on port 443 as an unprivileged user`_
+below. In principle something similar to this should work on a Linux/Unix
+system::
+
+    $ sudo useradd relaysrv
+    $ sudo mkdir /etc/relaysrv
+    $ sudo chown relaysrv /etc/relaysrv
+    $ sudo -u relaysrv /usr/local/bin/relaysrv -keys /etc/relaysrv
+
+This creates a user ``relaysrv`` and a directory ``/etc/relaysrv`` to store
+the keys. The keys are generated on first startup. The relay will join the
+global relay pool, unless a ``-pools=""`` argument is given.
+
+To make the relay server start automatically at boot, use the recommended
+procedure for your operating system.
+
+Running on port 443 as an unprivileged user
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is recommended that you run the relay on port 443 (or another port which is
+commonly allowed through corporate firewalls), in order to maximise the chances
+that people are able to connect. However, binding to ports below 1024 requires
+root privileges, and running a relay as root is not recommended. Thankfully
+there are a couple of approaches available to you.
+
+One option is to run the relay on port 22067, and use an ``iptables`` rule
+to forward traffic from port 443 to port 22067, for example::
+
+    iptables -t nat -A PREROUTING -i eth0 -p tcp --dport 443 -j REDIRECT --to-port 22067
+
+Or, if you're using ``ufw``, add the following to ``/etc/ufw/before.rules``::
+
+    *nat
+    :PREROUTING ACCEPT [0:0]
+    :POSTROUTING ACCEPT [0:0]
+
+    -A PREROUTING -i eth0 -p tcp --dport 443 -j REDIRECT --to-port 22067
+
+    COMMIT
+
+You will need to start ``relaysrv`` with ``-ext-address ":443"``. This tells
+``relaysrv`` that it can be contacted on port 443, even though it is listening
+on port 22067. You will also need to let both port 443 and 22067 through your
+firewall.
+
+Another option is `described here <https://wiki.apache.org/httpd/NonRootPortBinding>`__,
+although your milage may vary.
+
+Recommended Specifications
+--------------------------
+
+To run a relay of your own, you will first need a server. Running a relay from
+a home computer is  not recommended: the relay ideally needs to be running 24/7,
+and home internet connections often have poor upload bandwidth.
+
+The recommended specifications for a relay are:
```

Comment:
I had put it at the bottom for this very reason, but now that you let me remove it I will :)